internal/server/audit/audit.go

```diff
@@ -18,7 +18,7 @@ const (
 	eventMetadataActionKey = "flipt.event.metadata.action"
 	eventMetadataTypeKey   = "flipt.event.metadata.type"
 	eventMetadataIPKey     = "flipt.event.metadata.ip"
-	eventMetadataAuthorKey = "flipt.event.metadata.author"
+	eventMetadataActorKey  = "flipt.event.metadata.actor"
 	eventPayloadKey        = "flipt.event.payload"
```

Comment:
regarding future proofing. wdy think about changing this to making `actor` an 'object' in case we have a different way of identifying them in the future (non-email), also potentially moving `ip` under `actor` so its more clear that its the IP address of the actor and not of the Flipt instance that's running?

so it would be something like 

`flipt.event.metadata.actor.email` and `flipt.event.metadata.actor.ip`
? 

/cc @yquansah

Reply:
Yeah I think this makes sense, since both of those properties are related to an actor. I wonder if it even makes sense to nest that under metadata?